src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.ts

```diff
@@ -95,6 +97,19 @@ export class ChatMessageEditor {
         this.notifyPanelObserver.observe(this.notifyPanel, {
             attributes: true,
         });
+
+        this.sideNavObserver = new MutationObserver(this.updateEditorFocus);
+        this.sideNavs = document.querySelectorAll('.side-nav');
+        if (ScreenSize.isNarrow()) {
```

Comment:
I think you can skip subscription for iOS devices, because blur does nothing in case of iOS device.

Reply:
Ok